pkgs/applications/version-management/git-and-tools/git-team/default.nix

```diff
@@ -0,0 +1,32 @@
+{ lib, buildGoModule, fetchFromGitHub, installShellFiles }:
+buildGoModule rec {
+  pname = "git-team";
+  version = "1.7.0";
+
+  src = fetchFromGitHub {
+    owner = "hekmekk";
+    repo = "git-team";
+    rev = "v${version}";
+    sha256 = "0nl5j64b61jw4bkf29y51svjbndmqqrqx96yaip4vjzj2dx9ywm4";
+  };
+
+  vendorSha256 = "sha256-xJMWPDuqoNtCCUnKuUvwlYztyrej1uZttC0NsDvYnXI=";
+
+  nativeBuildInputs = [ installShellFiles ];
+
+  postInstall = ''
+    go run main.go --generate-man-page > $out/${pname}.1
+    installManPage $out/${pname}.1
+
+    go run main.go completion bash > $out/${pname}.bash
+    installShellCompletion --bash --name ${pname}.bash $out/${pname}.bash
```

Comment:
Maybe add a comment to remind future people to check if newer version of upstream have added completion for [other shells](https://github.com/nixos/nixpkgs/blob/1d3edd77efd72a36ce2508d3ba19de3c35452483/pkgs/build-support/setup-hooks/install-shell-files.sh#L81)?

Reply:
I work in the same company as git-team's author and there's no plan to add more completions for years now. I'd rather call for yagni, but I added a comment for clarification anyways.